step.sh

```diff
@@ -9,10 +9,11 @@ fi
 #Setup env vars
 LANG="${lang}"
 FILENAME_TO_UPLOAD="${huawei_filename}"
+FILE_EXT="${apk_path##*.}"
 
 printf "Apk path is: ${apk_path}\n"
```

Comment:
`apk_path `should be renamed to `file_path` and log message should be **File path** instead of **Apk path**